packages/cursorless-engine/index.ts

```diff
@@ -0,0 +1,11 @@
+export * from "./testUtil/plainObjectToTarget";
```

Comment:
we should have a look at these exports as we get more serious about exposing cursorless engine as an npm package, eg for rango or embedding in MS Word